Tasks/UseDotNetV2/models.ts

```diff
@@ -113,34 +115,6 @@ export class VersionParts {
         }
     }
 
-    /**
-     * Validate the version if this string is a explicit version number. Returns an exception if the version number is not explicit.
-     * @param version the input version number as string
-     */
-    private static ValidateExplicitVersionNumber(version: string): void {
```

Comment:
Why are you removing this functionality. it checks whether user has entered an explicit version or not